installer/manifests/keptn/charts/control-plane/values.yaml

```diff
@@ -81,6 +81,8 @@ apiService:
   config:
     maxAuthRequestsPerSecond: "1.0"
     maxAuthRequestBurst: "2"
+  features:
```

Comment:
given that this URL will ultimately also be used in the shipyard controller, should we move that value up one level (e.g. `controlPlane.features.automaticProvisioningURL`)?